src/Microsoft.Azure.WebJobs.Host/Executors/FunctionExecutor.cs

```diff
@@ -537,8 +545,24 @@ private static ITaskSeriesTimer StartParameterLogTimer(IRecurrentCommand updateC
             //      a. If throwOnTimeout, we throw the FunctionTimeoutException.
             //      b. If !throwOnTimeout, wait for the task to complete.
 
-            // Start the invokeTask.
-            Task<object> invokeTask = invoker.InvokeAsync(invokeParameters);
+            // Check to see if there are any filters to decide which invoke method to call
+            List<IFunctionInvocationFilter> filters = new List<IFunctionInvocationFilter>();
+            filters = GetFilters(instance.FunctionDescriptor, invoker.Instance);
+
+            // Prepare the invokeTask result
+            Task<object> invokeTask = null;
+
+            if (filters.Any())
+            {
+                // Because there are filters, invoke the function with the filters
+                invokeTask = InvokeWithFiltersAsync(filters, invoker, parameterHelper, invokeParameters, functionCancellationTokenSource, instance,
```

Comment:
It's utilized for the original implementation of invoking the function, InvokeAsync on the invoker inside that InvokeWithFiltersAsync method